mopro-ffi/src/app_config/android.rs

```diff
@@ -116,3 +114,25 @@ fn move_bindings(bindings_out: &Path, bindings_dest: &Path) {
     }
     fs::rename(bindings_out, bindings_dest).expect("Failed to move bindings into place");
 }
+
+fn generate_android_bindings(dylib_path: &Path, binding_dir: &Path) -> Result<(), Error> {
```

Comment:
I think with `use anyhow::Result;`
we don't have to define `Error`
just use `Result<()>`?